### DISCUSSION TITLE
How to maintain the order of CSS declarations?

### DISCUSSION THREAD

**Discussion by phongphongg** [chakra-ui/panda#2508] [Q&A]
Is there a way to maintain the order of CSS declarations?

Link: https://play.panda-css.com/GACxJoJjMd

Even though I declared _pressed after _hover, after building, it becomes _pressed before _hover. This causes the style of _pressed to be overridden.

<img width="579" alt="image" src="https://github.com/chakra-ui/panda/assets/58473133/b384dd10-95a7-4172-a7e4-d7b7f66ef2b8">

**Comment by koba04** [comment]
I have the same issue and had to define `_pressed` inside `_hover` as well.

```jsx
css({
  color: {
    _hover: {
      base: 'red'
      _pressed: 'blue'
    },
    _pressed: 'blue'
  }
}
```

**Comment by kysley** [comment]
Being able to use condition shortcuts in arbitrary queries would help 👀 `&:not(_pressed)` as an example
